src/libraries/Microsoft.PowerFx.Core/Glue/SimpleResolver.cs

```diff
@@ -15,184 +15,16 @@
 
 namespace Microsoft.PowerFx.Core.Glue
 {
-    /// <summary>
-    /// Basic implementation of INameResolver around a <see cref="PowerFxConfig"/> object. 
-    /// This aides in binding and intellisense. 
-    /// Host can override Lookup to provide additional symbols to the expression. 
-    /// </summary>
-    internal class SimpleResolver : INameResolver
+    [Obsolete("Use symbol table")]
+    internal class SimpleResolver : ComposedReadOnlySymbolTable
```

Comment:
This is to avoid breaking changes to Dataverse, yes?
Do we have a task to clean up the SQL target's uses and then remove these Obsolete members?

Reply:
Right. 
I can file a task.